tests/functional/example/conftest.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Common fixtures and utils for example tests."""
+
+import os
+import tempfile
+
+import pytest
+
+
+@pytest.fixture(scope="session")
+def setup_database():
+    """Configure the database"""
+    os.environ['ORION_DB_TYPE'] = "pickleddb"
+    os.environ['ORION_DB_ADDRESS'] = tempfile.NamedTemporaryFile().name
```

Comment:
Maybe we should handle  `tmp_file.close()` cleanly here. I don't know if using yield could work if the fixture is not used explicitly as an argument to the test function. You would need to verify.

Reply:
The file is deleted automatically at the end of the session though.
I confirm that yield works even when not used explicitly :+1: .